sample-apps/payment-customization-sample-app/web/frontend/components/CustomizationForm.jsx

```diff
@@ -42,7 +44,14 @@ export function CustomizationForm({
           />
         </FormLayout.Group>
 
-        <Button submit primary loading={loading}>Save Customization</Button>
+        <Button
+          submit
+          primary
+          loading={loading}
+          disabled={!isNewCustomization && !hasChanged}
+        >
+          Save Customization
+        </Button>
```

Comment:
I would prefer this button to be a [Page Action](https://polaris.shopify.com/components/page-actions)